mibidata/tiff.py

```diff
@@ -64,30 +64,60 @@ def write(filename, image, sed=None, optical=None, ranges=None,
         multichannel: Boolean for whether to create a single multi-channel TIFF,
             or a folder of single-channel TIFFs. Defaults to True; if False,
             the sed and optical options are ignored.
-        write_float: If True, saves the image data as float32 values (for
-            opening properly in certain software such as Halo). Defaults to
-            False which will save the image data as uint16. Note: setting
-            write_float to True does not normalize or scale the data before
-            saving, however saves the integer counts as floating point numbers.
+        dtype: Forces the image data saved as either float or uint16. Can
+            specify or ``np.float32`` to force data to be saved as floating
+            point values or ``np.uint16`` to save as unsigned 16-bit integer
+            values. Defaults to None, which saves data as its original type, if
+            the original type can converted without a loss of data.
+        write_float: Deprecated, will raise ValueError if specified. To
+            specify the dtype of the saved image, please use the `dtype`
+            argument instead.
 
     Raises:
-        ValueError: Raised if the image is not a
-            ``mibitof.mibi_image.MibiImage`` instance, or if its coordinates,
-            size, masses or targets are None.
+        ValueError: Raised if
+
+            * The image is not a :class:`mibidata.mibi_image.MibiImage` instance
+            * The :class:`mibidata.mibi_image.MibiImage` coordinates, size,
+              masses or targets are None
+            * `dtype` is not one of ``np.float32`` or ``np.uint16``
+            * `write_float` has been specified.
+            * Converting the native :class:`mibidata.mibi_image.MibiImage` dtype
+              to the specified or inferred ``dtype`` results in a loss of data.
```

Comment:
Please be consistent with whether you end your items with `.` or not

Reply:
I am embarrassed for myself...